sourcefiles/randomizer.py

```diff
@@ -1191,6 +1192,9 @@ def summarize_single(choicelist):
         return rv
 
     def write_settings_spoilers(self, file_object):
+        if self.hash_string_bytes is not None:
+            hashstr = str(ctstrings.CTNameString(self.hash_string_bytes)).replace('*', '{star}')
```

Comment:
I figured using `{star}` was nicer here than using the `*` since every other character that appears in seed hashes is done with the bracketed version. For the webgen, I'm planning to put some actual images on the page as well (and of course for the seed image too).